README.md

```diff
@@ -792,21 +782,50 @@ See [openhab-js : actions.Voice](https://openhab.github.io/openhab-js/actions.ht
 
 #### Cloud Notification Actions
 
-Note: Optional action if [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/) is installed.
+Requires the [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/) to be installed.
 
 Notification actions may be placed in rules to send alerts to mobile devices registered with an [openHAB Cloud instance](https://github.com/openhab/openhab-cloud) such as [myopenHAB.org](https://myopenhab.org/).
 
-For available actions have a look at the [Cloud Notification Actions Docs](https://www.openhab.org/docs/configuration/actions.html#cloud-notification-actions).
+There are three different types of notifications:
+
+- Broadcast Notifications: Sent to all registered devices and shown as notification on these devices.
+- Log Notifications: Only shown in the notification log, e.g. inside the Android and iOS Apps.
+- Standard Notifications: Sent to the registered devices of the specified user and shown as notification on his devices.
+
+To send these three types of notifications, two notification builders are available:
+
+- `.logNotificationBuilder(message)`: Creates a builder to send a log notification with the given message.
+  - `.withIcon(icon)`: Sets the icon of the notification.
+  - `.withSeverity(link)`: Sets the severity of the notification.
+  - .`send()`: Sends the notification.
```

Comment:
I'd replace this builder with a function `logOnly()` on `notificationBuilder()`. This way the media attachment can be shown in the notification list.

Reply:
Good idea!

Reply:
But please note that media attachment cannot be set for log notifications with the current NotificationActions in https://github.com/openhab/openhab-addons/issues/16934.

Reply:
@digitaldan Maybe drop the three user-facing types (log, broadcast, normal) completely and make everything a "normal" notification? Then log notifications are just like normal ones with the same properties, but don't push.